components/feature/addons/src/main/java/mozilla/components/feature/addons/ui/AddonInstallationDialogFragment.kt

```diff
@@ -145,13 +142,11 @@ class AddonInstallationDialogFragment : AppCompatDialogFragment() {
 
     @SuppressLint("InflateParams")
     private fun createContainer(): View {
-        val rootView = LayoutInflater.from(requireContext()).inflate(
-            R.layout.mozac_feature_addons_fragment_dialog_addon_installed,
-            null,
-            false
+        val binding = MozacFeatureAddonsFragmentDialogAddonInstalledBinding.inflate(
+            LayoutInflater.from(requireContext()), null, false
```

Comment:
I don't think it's worth introducing view binding just for this here since we could look up those views manually here?

Reply:
I took the opportunity here to migrate to view binding to streamline view lookups. As I think it makes sense to have all lookups migrated in future. But I can of course revert and change the synthetic lookup here to a manual one if you think it isn't worth or is overkill.